projects/angular/src/popover/tooltip/tooltip-content.ts

```diff
@@ -102,9 +104,7 @@ export class ClrTooltipContent extends AbstractPopover {
         this.popoverPoint = Point.RIGHT_TOP;
         break;
       default:
-        this.anchorPoint = Point.RIGHT_CENTER;
-        this.popoverPoint = Point.LEFT_TOP;
-        break;
+        assertNever(newPosition);
```

Comment:
Since the parameter to `assertNever` is typed as `never`, there will be TypeScript compile error if there is a missing `case`.